### ISSUE TITLE
SNMP custom endpoints

### ISSUE THREAD

**Issue by jussisjostrom** [F5Networks/f5-telemetry-streaming#207]
### Environment
 * Telemetry Streaming Version: 1.29
 * BIG-IP Version: 15.1.6

### Summary
It seems custom metrics obtained using SNMP works only if the type is Counter. If the type is INTEGER  then the value of the metric is not extracted correctly.
The sysStatMemoryUsed metric from the provided example works just fine, but if I try to report sysChassisPowerSupplyStatus, no report is sent for the metric.

### Steps To Reproduce
1. Define the custom endpoint like this:
```json
{
...
    "items": {
        "psuStates": {
            "name": "psuStates",
            "path": "1.3.6.1.4.1.3375.2.1.3.2.2.2.1.2",
            "protocol": "snmp"
        },
...
}
```

2. Observe on the collector, no report on metric psuStates...

When snmp-walking a working OID (eg sysStatMemoryUsed) output looks like this
```
# snmpwalk  -v2c -c public localhost  1.3.6.1.4.1.3375.2.1.1.2.1.45
F5-BIGIP-SYSTEM-MIB::sysStatMemoryUsed.0 = Counter64: 6179646608
```
And a custom endpoint defined using above OID is reported correctly.

For the PSU states, output looks like this:
```
# snmpwalk  -v2c -c public localhost 1.3.6.1.4.1.3375.2.1.3.2.2.2.1.2
F5-BIGIP-SYSTEM-MIB::sysChassisPowerSupplyStatus.1 = INTEGER: good(1)
F5-BIGIP-SYSTEM-MIB::sysChassisPowerSupplyStatus.2 = INTEGER: good(1)
F5-BIGIP-SYSTEM-MIB::sysChassisPowerSupplyStatus.3 = INTEGER: bad(0)
F5-BIGIP-SYSTEM-MIB::sysChassisPowerSupplyStatus.4 = INTEGER: bad(0)
```
So, maybe the parsing of the response from SNMP needs improvement so it works also with INTEGER Gauges? 

### Expected Behavior
sysChassisPowerSupplyStatus should be reported by TSE. After all, it is a numeric value (0, or 1), so it should be possible to send to StatsD consumer.

### Actual Behavior
No metric is reported from the sysChassisPowerSupplyStatus OID. 
If I configure the endpoint to grab the table sysChassisPowerSupplyTable and configure my StatsD consumer with "addTags method sibling", I get the sysChassisPowerSupplyStatus as tags, but not as values.

**Comment by dstokesf5** [comment]
Thank you for your feedback. I have added this to our internal product backlog as TS-668.

**Comment by dstokesf5** [comment]
This has been resolved in the 1.30.0 release of Telemetry Streaming.
